src/server/ua_server.c

```diff
@@ -15,10 +15,16 @@
  *    Copyright 2017 (c) Mark Giraud, Fraunhofer IOSB
  *    Copyright 2018 (c) Hilscher Gesellschaft für Systemautomation mbH (Author: Martin Lang)
  *    Copyright 2019 (c) Kalycito Infotech Private Limited
+ *    Copyright (c) 2020 Yannick Wallerer, Siemens AG
+ *    Copyright (c) 2020 Thomas Fischer, Siemens AG
  */
 
 #include "ua_server_internal.h"
 
+#ifdef UA_ENABLE_PUBSUB_CONFIG
+#include "ua_pubsub_config.h"
+#endif
+
```

Comment:
The include above (ua_server_internal.h) includes internally the "ua_pubsub_manager.h". The ua_server.c file very central and I would prefer to move your changes to the pubsub_manager. (attach to pubsub manager delete and so on). I think the ua_server.c file must not be changed by this PR.